fr.openfisca.org/api/v22/openfisca-web-api-fr22.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=OpenFisca Web API - New
+
+[Service]
+EnvironmentFile=/home/openfisca/secret/piwik_token
+Environment=SERVER_NAME=fr.openfisca.org/api/v22
+ExecStart=/home/openfisca/virtualenvs/api-fr22/bin/openfisca serve --port 8022 --workers 2 --tracker-idsite 4 --tracker-url https://stats.data.gouv.fr/piwik.php --country-package openfisca_france --tracker-token $TRACKER_TOKEN --welcome-message "Welcome to the OpenFisca-France Web API. To learn how to use it, check our interactive swagger documentation: https://fr.openfisca.org/legislation/swagger."
```

Comment:
As long as the legExp is on 21, I would keep 2 there.
I reduces the workers on 18 to 1.

Reply:
Ok, let's get the v22 online, but the next thing to do should be to migrate the legislation explorer to v22, in the spirit of continuous deploymente

Reply:
(And we also need to adapt our deployment of the leg exp so that it can deals with the new config stuff)

Reply:
agreed. I want to take some time to do it because there is the new .env configuration. As it will impact ops, I will increase/decrease the workers at the same time.